HelpSource/Guides/WritingHelp.schelp

```diff
@@ -338,4 +310,48 @@ The easiest way to contribute to the documentation is:
     code::
     git push origin -u doc_updates
     ::
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
you haven't properly resolved merge conflicts here -- please edit this document until it looks how you intend it.

Reply:
btw, i left the instructions on the wiki for this vague bc i don't know of an easy way to explain merge conflicts and how to resolve them, but i just found that github has some pretty clear documentation on it here: https://docs.github.com/en/github/using-git/resolving-merge-conflicts-after-a-git-rebase. i've linked that in the corresponding wiki section. hope that helps.

Reply:
This should be solved now I think.